### PR TITLE
feat: document headers list

### PR DESCRIPTION
**Pull request by samudra-ajri**
## Overview
This feature will be consumed by Sidebar v2 to sycnchronize the "kop naskah".

## Links
Task: [Clickup](https://app.clickup.com/t/860pkxbcf)

## Evidence
title: feat: document headers list
project: SIDEBAR
participants: @samudra-ajri @indraprasetya154 @fajarhikmal214 
screenshot: https://user-images.githubusercontent.com/54126488/215365121-d8bc4ce8-0f82-49a1-aaa8-8dc3f3078680.png

### CODE REVIEW

**Review by fajarhikmal214 — APPROVED** [review]
LGTM

File: src/app/Http/Controllers/V1/DocumentHeadersController.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     * Handle the incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function __invoke(Request $request)
+    {
+        $records = DocumentHeaderView::query();
+        $size = $request->input('size');
+        $records->join('mapping_per_pd', 'mapping_per_pd.RoleId', '=', 'v_kopnaskah.RoleId')
```
Comment (indraprasetya154) [inline src/app/Http/Controllers/V1/DocumentHeadersController.php:22]: is this a new table for sync to id organization in v2?

Comment (samudra-ajri) [reply src/app/Http/Controllers/V1/DocumentHeadersController.php:22]: Nope, this table is from the existing one. 😁

Comment (indraprasetya154) [reply src/app/Http/Controllers/V1/DocumentHeadersController.php:22]: whoooo nice information, thanks for an information yes ❤️

**Pull request merged**